src/Components/signin/signin-form/index.js

```diff
@@ -1,9 +1,9 @@
-import React from 'react';
-import StandardInput from '../../../Atoms/Forms/TextInputField';
-import ModuletteButton from '../../../Atoms/StandardButton';
-import styled from 'styled-components';
+import React from './node_modules/react';
+import StandardInput from '../../../atoms/forms/textinput-field';
+import ModuletteButton from '../../../atoms/standard-button';
+import styled from './node_modules/styled-components';
 import crypto from '../../../utils/crypto';
-import axios from 'axios';
+import axios from './node_modules/axios';
```

Comment:
Remove `./node_modules/` since a folder called `node_modules` does not exist in the SigninForm folder. Or use full path to root `node_modules`.